testing/unit_tests/t_utils.py

```diff
@@ -38,3 +49,19 @@ def create_tmp_dir(copy_data_testing_dir=True):
     if Path(path_data_testing_source).exists() and copy_data_testing_dir:
         shutil.copytree(path_data_testing_source,
                         path_data_testing_tmp)
+
+
+def create_tmp_dir_multi_session():
+    """Create a temporary directory for data related to multi-session unit tests and copy test data files.
+    1. Remove the ``tmp`` directory if it exists.
+    2. Copy the ``data_testing_multi`` directory to the ``tmp`` directory. Ignoring the `.git` folder within
+    Any data files created during testing will go into ``tmp`` directory.
+    This is created/removed for each test.
+    """
+    ignore_git_pattern = shutil.ignore_patterns(str(path_data_multi_sessions_contrasts_source / '.git'))
+    remove_tmp_dir()
+    os.mkdir(path_temp)
+    if os.path.exists(path_data_multi_sessions_contrasts_source):
```

Comment:
Here we use "os" whereas in `create_tmp_dir` above, we use "Path". Is there a preferred method? I would suggest using the same if possible.

Reply:
Great point. I must woke on the wrong  side of the bed for this. Fixed to be consistent with Pathlib.

Reply:
Hey @dyt811, did you push your changes? I see you wrote "fixed" for most of my comments but still see the code version before the fix, am I missing something?